doc/part-1-github-ui/github-contributions.md

```diff
@@ -8,6 +8,27 @@ description: 'PART 1: GITHUB UI'
 
 At this point, the focus is not about Git and Github best practices. The learning videos here will cover the basic things needed to get started with contributing to any project using the Github UI. What you will learn will also help you to tackle the tasks that have been provided for you to do after watching the learning videos.
 
+You will learn the following in this section:
+
+1. Forking a repository and how to know which branch you are in. 
+   
+2. Contributions (overview, issues, commits) and branching. 
```

Comment:
@aaryansr26 I mentioned using unordered list. Please help change it.

Reply:
Done! @Ifycode